src/components/Send.jsx

```diff
@@ -113,6 +113,29 @@ const CollaboratorsSelector = ({ numCollaborators, setNumCollaborators, minNumCo
   )
 }
 
+const enhanceTakerErrorMessageIfPossible = async (requestContext, httpStatus, errorMessage) => {
+  const configExists = (section, field) => Api.postConfigGet(requestContext, { section, field }).then((res) => res.ok)
+
+  const isConfigVarError = httpStatus === 409 && errorMessage === 'Action cannot be performed, config vars are not set.'
```

Comment:
Checking for an exact match with the error message might be a bit tricky as those can probably change without warning. I understand that there's probably no better way to do it. Just wanted to bring it up.

Reply:
Yeah, thinking the same. Do you think we should test for the config vars on any `409` error and just append the custom message?

Reply:
Done.